File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.java
class Solution {
    public long solution(int price, int money, int count) {
        long total = 0;
        for (int i = 1; i < count + 1; i++) {
            total += price * i;
        }

        return total - money <= 0 ? 0 : total - money;
    }
}

File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.js
function solution(price, money, count) {
    var answer = -1;
    var i = 0;
    var total = 0;
    while (true){
        i+=1;
        if (i>count) break;
        total += price * i;
    }
    answer = total - money;
    if (answer < 0) answer = 0;

    return answer;
}